test/cli_start_test.go

```diff
@@ -233,3 +233,14 @@ func (suite *PouchStartSuite) TestStartWithPrivilege(c *check.C) {
 
 	command.PouchRun("start", name).Assert(c, icmd.Success)
 }
+
+// TestStartWithAnnotation starts a container with annotation.
+func (suite *PouchStartSuite) TestStartWithAnnotation(c *check.C) {
+	name := "start-annotation"
+
+	res := command.PouchRun("create", "--name", name, "--annotation", "a=b", busyboxImage)
+	res.Assert(c, icmd.Success)
+	defer DelContainerForceMultyTime(c, name)
+
+	command.PouchRun("start", name).Assert(c, icmd.Success)
```

Comment:
I think after start, we need to check these annotation as well with `pouch inspect`.

Reply:
this has been done in `test/cli_run_test.go`

Reply:
No, I do not think they are the same. They have called the same API, but in cli side, they called different code. Double check would be nicer to guarantee the code quality.

Reply:
Follow the previous logic, inspect test in doing in cli_run_test.go, not in start test, as I agree with you, this also related all test case in `test/cli_start_test.go`.

Reply:
oh! I miss create test, this things should be done there too.